### PR TITLE
Convert demo readme to markdown

### PR DESCRIPTION
**Pull request by tammyleino**
Markdown format required for OpenAMP doc generation.
Signed-off-by: Tammy Leino <tammy_leino@mentor.com>

### CODE REVIEW

File: demos/openamp/README.md [hunk 1]
```diff
@@ -1,66 +1,64 @@
...
+    humanfriendly = "*"
+```
+
+## 2) Change into the lopper demo directory
+
+```
+    % cd demos/openamp
+```
+
+## 3) Execute Lopper with openamp assets and lops
```
Comment (tammyleino) [inline demos/openamp/README.md:37]: @zeddii Kindly confirm whether "assits" in the original file is correct.

Comment (wmamills) [reply demos/openamp/README.md:37]: Yes, they "assist" with the conversion.  I am no speller but I think that it is OK from what I looked up.

Comment (zeddii) [reply demos/openamp/README.md:37]: Sorry was away from my keyboard. @wmamills is correct, that should have been "assists"

Comment (tammyleino) [reply demos/openamp/README.md]: @wmamills @zeddii  Thank you for feedback - fixed.

**Pull request merged**